modules/template/src/main/scala/scaladex/view/Task.scala

```diff
@@ -24,6 +24,11 @@ object Task {
     "Update the Github info of an existing project"
   )
 
+  val updateMavenArtifacts: Task = Task(
+    "update-maven-artifact",
+    "Update existing artifacts with new fields"
```

Comment:
To be a bit more explicit:
```suggestion
    "Download all pom files to update existing artifacts with new fields"
```